pkgs/development/libraries/nrf5-sdk/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nrf5-sdk";
+  version = "17.1.0";
+
+  urlHash = "ddde560";
+
+  src = fetchurl {
```

Comment:
I believe that if you use fetchzip instead you won’t have to bother with unpacking it manually

Reply:
I tried doing that, but it failed with `do not know how to unpack source archive`.

Reply:
I've written this [small patch](http://0x0.st/oSBw.txt) to fix it

Reply:
Thabk you! I will apply it later today.

Reply:
@legendofmiracles You patch appears to function correctly. I suspect I had some weird caching or network stability issues before.